com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/MotionVectorVertexShaderCommon.hlsl

```diff
@@ -117,9 +132,13 @@ PackedVaryingsType MotionVectorVS(inout VaryingsType varyingsType, AttributesMes
         AttributesMesh previousMesh = inputMesh;
         previousMesh.positionOS = effectivePositionOS;
 
-        previousMesh = ApplyMeshModification(previousMesh, _LastTimeParameters.xyz
+        previousMesh = ApplyMeshModification(previousMesh,
+            _LastTimeParameters.xyz
     #if defined(USE_CUSTOMINTERP_APPLYMESHMOD)
             , varyingsType.vmesh
+    #endif
+    #if defined(HAVE_VFX_MODIFICATION)
+            , element
```

Comment:
@julienf-unity The attributes must be passed to the previous frame's vertex stack